concepts/observability-strategy/configurable-logging/01-logging-groundwork.md

```diff
@@ -0,0 +1,47 @@
+# Configurable Logging: Groundwork
+
+## Current Situation and Motivation
+
+![a](./assets/logging-current.drawio.svg)
+
+In the current setup, the logging component provides feature-rich e2e logging solution with Loki as backend and Grafana as log Browser. The used log collector (FluentBit) can be configured flexibly and neutrally. However, users must apply the configuration during installation; otherwise it's lost at the next Kyma upgrade process. Furthermore, it is hard to mix and match different integrations, as you must deal with one centralized configuration (the Fluent Bit config) at deploy time. Furthermore, Loki is provided in a non-scalabale setup and cannot configured at runtime.
+
+As outlined in the [general strategy](../strategy.md), integration (and with that, changing the focus away from in-cluster backends) is the key to open up the stack for a broad range of use cases. Users can simply bring their own backends if they already use a commercial offering or run their own infrastructure. The data can be stored outside the cluster in a managed offering, shared with the data of multiple clusters, away from any tampering or deletion attempt of a hacker, to name just a few.
+
+This concept proposes how to open up to those new scenarios by supporting convenient integration at runtime.
+
+## Requirements
+
+### Basic backend configuration
+- Have a vendor-neutral agent layer that collects and ships the telemetry data, but does not permanently store it (as a backend)
```

Comment:
didn't we want to throw out the term "agent layer" ?

Reply:
To follow up on our brief call this morning: "Agent layer" is not wrong, it's a perfectly fine term. But if we want to use it, we need to define the term, and then it maybe should also show up in the diagrams. 
Alternatively, we can decide that the term isn't need it and just say "collectors", "collector layer" or something like that; sticking to words we're already using.

Reply:
fixed